README.md

```diff
@@ -36,6 +38,7 @@ cd url-shortener-1
 Step 4. Install the node dependencies.
 ```sh
 npm install
+node app.js
```

Comment:
Why is that?

Reply:
Seems like a merge error. I just fixed it and pushed the update